source/IAccessibleHandler/internalWinEventHandler.py

```diff
@@ -211,3 +244,25 @@ def _shouldGetEvents():
 # winEventCallback adds these whenever it sees an event for ConsoleWindowClass windows,
 # As winEvents always contain the true thread ID.
 consoleWindowsToThreadIDs: Dict[int, int] = {}
+
+
+def getNVDAObjectsToMonitor():
+	import api
+	objs = [
+		*api.getFocusAncestors(),
+		api.getFocusObject(),
+		# The desktop object should be in the focus ancestry, but include it here just
+		# in case.
+		api.getDesktopObject()

```

Comment:
Are there cases where the desktop object isn't in the ancestry?

I think it might make sense to make this a set. While not waterproof, it could avoid duplicates in here.